client/src/components/TestSuite/TestSuiteTree/TestSuiteTree.tsx

```diff
@@ -76,6 +84,17 @@ const TestSuiteTreeComponent: FC<TestSuiteTreeProps> = ({
           />
           <Divider />
           {testGroupList}
+          <Divider />
```

Comment:
Note that this looks a little weird when the last actual item has no text -- not an actual problem with this PR